src/aind_data_schema_models/_generators/generator.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+from jinja2 import Environment
+import pandas as pd
+from aind_data_schema_models.utils import to_class_name, to_class_name_underscored
+from pathlib import Path
+import subprocess
+
+
+def generate_code(data_type: str, isort: bool = True, black: bool = True):
+    """Generate code from the template type
+
+    Parameters
+    ----------
+    data_type : str
+        Which template file to use
+    isort : bool, optional
+        Whether to run isort on the output, by default True
+    black : bool, optional
+        Whether to run black on the output, by default True
+    """
+    ROOT_DIR = Path("./src/aind_data_schema_models/")
```

Comment:
Can we make the input path configurable as an input arg?

Reply:
added